android/src/main/java/com/tailscale/ipn/App.java

```diff
@@ -86,17 +88,26 @@ public class App extends Application {
 
 	public DnsConfig dns = new DnsConfig(this);
 	public DnsConfig getDnsConfigObj() { return this.dns; }
+	public AppsConfig acfg = null;
 
 	@Override public void onCreate() {
 		super.onCreate();
+		Log.v("com.tailscale.ipn", "Loading apps: " + System.currentTimeMillis()/1000L);
+		try {
+			acfg = new AppsConfig(this, getEncryptedPrefs());
+			//acfg.printConfig();
```

Comment:
Yes I left them there because I was worried of the impact in the starting time on the App depending on the number of apps, as you pointed below loading the apps at the start is far from optimal.